src/cljs/zetawar/subs.cljs

```diff
@@ -220,6 +220,14 @@
                        conn @(current-faction-eid conn)))
       0))
 
+(deftrack faction-color-name [faction]
+  (when @faction
+    (-> @faction
+        :faction/color
+        name
+        (#(str % "-name"))
```

Comment:
I think you can eliminate the anonymous function altogether can't you? In other words, turn '(#(str % "-name"))" into '(str "-name")'.